README.md

```diff
@@ -98,144 +95,123 @@ different element _lifecycle events_:
    attributes unique to your element.
 
 ## Browser Support
-**Don't let the fact that the [Custom Elements specification][spec] is a
-working draft scare you!**
-
-As of this writing, Chrome has already [implemented][Chrome] the [v0 spec],
-and both [Safari] and [Firefox] are working on their own implementations of
-[v1](#v1). Regardless of which browsers end up implementing which spec
-(if any), the v0 spec is "frozen" (meaning it will not change), and
-there are several solid [polyfills](#polyfills) that bring custom
-element support to just about every browser—including IE8!
+As of summer, 2018, Chrome has already [implemented][Chrome] both the [v0 spec],
+and the adopted ("v1") [spec]. [Safari] has implemented so-called "autonomous"
+custom elements, but has officially refused to support extension of built-in
+elements. [Firefox] has implemented the spec behind a feature flag. Regardless
+of your support targets, you should probably be using one of the [polyfills](#polyfills).
 
 That said, when using custom elements—or anything involving JavaScript,
 for that matter—**you should always design experiences for progressive
 enhancement**, and plan for the possibility that JavaScript isn't
-enabled or available.
+enabled or available. ✌️
 
-## The APIs
-
-### v0
-The working draft spec was formally "frozen" and dubbed [v0][v0 spec]
-in early 2016, and [several browsers][caniuse] have already shipped
-native implementations. The v0 API consists of a single function:
+## The API
+The custom elements API consists mainly of a "custom element registry" object
+that can be used to register class constructors for custom elements by name:
 
 ```js
-document.registerElement('element-name', ElementClass);
+window.customElements.define('element-name', ElementClass);
 ```
 
-Where `ElementClass` can either be a first-class constructor that extends
-[HTMLElement], or an object with a `prototype` that extends it:
+Where `ElementClass` is a class that extends [HTMLElement]:
 
 ```js
-// ES5
-var ElementClass = {
-  prototype: Object.create(
-    HTMLElement.prototype,
-    {
-      // descriptors here
-    }
-  )
-};
-
 // ES2015
 class ElementClass extends HTMLElement {
-  // methods and accessors here
+  constructor() {
+    // super() // <-- this is required!
```

Comment:
Maybe uncomment the super function call and document that it must be called first?